src/tests/test_utils/common_test_utils/include/common_test_utils/subgraph_builders/preprocess_builders.hpp

```diff
@@ -443,6 +443,25 @@ inline std::shared_ptr<Model> crop_dynamic() {
     return function;
 }
 
+inline std::shared_ptr<Model> pad_constant() {
+    using namespace ov::preprocess;
+    auto function = create_preprocess_1input(ov::element::f32, PartialShape{1, 3, 10, 10});
+    auto p = PrePostProcessor(function);
+    p.input().preprocess().pad({0, 0, 2, 2}, {0, 0, -1, 3}, 0, PaddingMode::CONSTANT);
+    function = p.build();
+    return function;
+}
+
+inline std::shared_ptr<Model> pad_edge() {
```

Comment:
Please put `pad_edge()` and `pad_constant` into `std::vector<preprocess_func> generic_preprocess_functions()` to make them can be tests.